src/test/java/com/radixdlt/serialization/mapper/RadixObjectMapperConfiguratorTest.java

```diff
@@ -0,0 +1,178 @@
+package com.radixdlt.serialization.mapper;

```

Comment:
Missed this one earlier.  Needs a header comment with copyright / license statement in it.

Reply:
Done